package-lock.json

```diff
@@ -1899,77 +1894,18 @@
         }
       }
     },
-    "@material-ui/core": {
-      "version": "4.11.0",
-      "resolved": "https://registry.npmjs.org/@material-ui/core/-/core-4.11.0.tgz",
-      "integrity": "sha512-bYo9uIub8wGhZySHqLQ833zi4ZML+XCBE1XwJ8EuUVSpTWWG57Pm+YugQToJNFsEyiKFhPh8DPD0bgupz8n01g==",
```

Comment:
There are a lot of removals from `package-lock.json`, which seems odd since we _added_ a new dependency to `package.json`.  Obviously not a blocker and not necessarily actionable, but seems possibly a bit strange.

Reply:
Let me take a look.

Reply:
Should look a lot better now.

Reply:
Yeah, _MUCH_ better.  Curious, what was the issue.  But in any event, _Resolved!_

Reply:
Mmm, probably switching back and forth between branches and maybe not doing the proper `npm install`. That's my guess.

Reply:
Yeah, makes sense.